lib/rmmz_windows/Window_NameEdit.d.ts

```diff
@@ -1,16 +1,218 @@
-import { Window_StatusBase } from "./Window_StatusBase";
-import { Rectangle } from "../rmmz_core/Rectangle";
-import { Game_Actor } from "../rpg_object/Game_Actor";
+import { Window_StatusBase } from ".";
+
+import { Rectangle, Sprite, Window } from "../rmmz_core";
+import { Game_Actor } from "../rmmz_objects";
+import { Window_Base } from "./Window_Base";
+import { Window_Help } from "./Window_Help";
 
 /**
  * Window_NameEdit
  * 
  * The window for editing an actor's name on the name input screen.
  */
-declare class Window_NameEdit extends Window_StatusBase {
+declare class Window_NameEdit implements Omit<Window_StatusBase, 'name' | keyof Window> {
```

Comment:
I hate this from the depths of my heart, but i couldn't find another way around it :d

The problem is PIXI defines a property `name` for its containers and the `name` method on this class was clashing with it.